xml/storage_multipath.xml

```diff
@@ -2489,723 +2406,682 @@ blacklist_exceptions {
+   <title>Setting aliases for multipath maps</title>
+   <para>
+    Arbitrary map names can be set in the <literal>multipaths</literal> section
+    of <filename>/etc/multipath.conf</filename> as follows:
+   </para>
+<screen>multipaths {
     multipath {
         wwid 3600a098000aad1e3000064e45f2c2355
         alias postgres
     }
 }</screen>
-    <para>
-      Aliases are expressive, but they need to be assigned to each map
-      individually, which may be cumbersome on large systems.
-    </para>
+   <para>
+    Aliases are expressive, but they need to be assigned to each map
+    individually, which may be cumbersome on large systems.
+   </para>
   </sect2>
 
   <sect2 xml:id="sec-multipath-user-friendly-names">
-    <title>Using autogenerated user-friendly names</title>
-    <para>
-      <systemitem>multipath-tools</systemitem> also supports autogenerated aliases,
-      so-called &ldquo;user-friendly names&rdquo;.
-      The naming scheme is follows: A name prefix (by default,
-      <literal>mpath</literal>) is followed by lower-case
-      letters. <literal>mpatha</literal> is the first possible name, followed by
-      <literal>mpathb</literal>, <literal>mpathc</literal>, …,
-      <literal>mpathz</literal>. After that follows
-      <literal>mpathaa</literal>, <literal>mpathab</literal>, and so on.
-    </para>
-    <para>
-      Map names are only useful if they are persistent.
-      <systemitem>multipath-tools</systemitem> keeps track of the assigned
-      names in the file <filename>/etc/multipath/bindings</filename> (the
-      &ldquo; bindings file&rdquo;).
-      When a new map is set up, the WWID is first looked up in this file.
-      If it is not found, the lowest available user-friendly name is assigned
-      to it.
-    </para>
-    <para>
-      Explicit aliases as discussed in <xref linkend="sec-multipath-alias"/>
-      take precedence over user-friendly names.
-    </para>
-    <para>
-      The following options in <filename>/etc/multipath.conf</filename> affect
-      user friendly names:
-    </para>
-    <variablelist>
-      <varlistentry>
-	<term>user_friendly_names</term>
-	<listitem>
-	  <para>
-	    If set to <literal>yes</literal>, user friendly names are
-	    assigned and used. Otherwise, the WWID is used as map name unless
-	    an alias is configured.
-	  </para>
-	</listitem>
-      </varlistentry>
-      <varlistentry>
-	<term>alias_prefix</term>
-	<listitem>
-	  <para>
-	    The prefix used to create user friendly names,
-	    <literal>mpath</literal> by default.
-	  </para>
-	</listitem>
-      </varlistentry>
-    </variablelist>
-  <warning>
+   <title>Using autogenerated user-friendly names</title>
+   <para>
+    <systemitem>multipath-tools</systemitem> also supports autogenerated
+    aliases, so-called &ldquo;user-friendly names&rdquo;. The naming scheme of
+    the aliases follows the pattern:
+    <replaceable>A_NAME_PREFIX_LOWER_CASE_LETTERS</replaceable>, for example,
```

Comment:
I don't like this much. Many readers will interpret this as a literal, and not realize that it's actually comprised of two parts (prefix and "number"). Is this a convention followed elsewhere in the SLE docs?

Reply:
The examples below may clarify the meaning,  but I still find `<replaceable>A_NAME_PREFIX_LOWER_CASE_LETTERS</replaceable>` confusing.

Reply:
Yes, this is stated even in the Preface of our docs that it is a placeholder and you need to replace it with our own value.

Reply:
My main point is that this obfuscates the fact that the placeholder consists of two independent parts. Actually, the underscore suggests (to me at least) that it's a single "variable". What about something like `<replaceable>PREFIX</replaceable>&nbsp;<replaceable>INDEX</replaceable>`, and explaining the two parts below?

Anyway, you're the boss, so if you want it this way, fair enough.